scripts/develop-module.js

```diff
@@ -0,0 +1,299 @@
+const fs = require('fs');
+const path = require('path');
+const arg = require('arg');
```

Comment:
Don't need to introduce another args package can you change this to use `yargs` which is what the other scripts use. It is already loaded in `package.json`
https://www.npmjs.com/package/yargs

Reply:
I did not know that we were using `yargs`. Will switch to it, thanks.